tests/lib/external/snapd-testing-tools/tools/tests.pkgs

```diff
@@ -19,6 +21,11 @@ cmd_install() {
     unsupported
 }
 
+cmd_install_local() {
```

Comment:
I do not see this defined anywhere, is this required? That am I missing?

Reply:
this code is imported from project https://github.com/snapcore/snapd-testing-tools
cmd_install_local is implemented in different scripts depending on the the tested system 

├── tests.pkgs
├── tests.pkgs.apt.sh
├── tests.pkgs.dnf-yum.sh
├── tests.pkgs.pacman.sh
├── tests.pkgs.zypper.sh